DSCResources/MSFT_xDatabaseLogin/MSFT_xDatabaseLogin.schema.mof

```diff
@@ -5,7 +5,6 @@ class MSFT_xDatabaseLogin : OMI_BaseResource
     [Write, ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
     [Key] String LoginName;
     [Write] String LoginPassword;
-    [Required, ValueMap{"Windows","SQL"}, Values{"Windows","SQL"}] string SqlAuthType;
```

Comment:
You're right. I hadn't considered it from a breaking change perspective. I will undo this change. The intention wasn't to remove the ability to support windows auth (if you read the psm1 it still uses it) but just to remove the need to expressly specify that you're. We can infer from the lack of SQL Auth credentials that you are using it. As a breaking change though that doesn't add anything I'll reverse it.